backend/routes/api/users.js

```diff
@@ -73,8 +88,7 @@ router.post(
       lastName: Joi.string().required(),
       oauthId: Joi.string().required(),
       propicUrl: Joi.string(),
-      isApproved: Joi.boolean().default(false),
-      role: Joi.string().required(),
+      role: Joi.string().default("PENDING"),
```

Comment:
Nit: can we use the enum value in place of string “PENDING” just in case we want to switch what that string is